### ISSUE TITLE
Did the Play store api change recently?

### ISSUE THREAD

**Issue by devzer01** [Lekensteyn/apk-downloader#12]
server:GSE
status:400 Bad Request
version:HTTP/1.1
x-content-type-options:nosniff
x-frame-options:SAMEORIGIN
x-xss-protection:1; mode=block

this is what i get when i tried today on a OSX chrome Version 34.0.1847.116

the last time i used this extension successfully  was about 3 weeks ago at work on a Ubuntu machine with the latest version of Chrome at that point. I am not sure if this is a local problem or Google change the API. 

i didn't post the form data here because i wasn't sure if it contained my password or not,

**Comment by cielavenir** [comment]
Perhaps logout -> login will fix that issue. Sometimes the token expires. I don't know why.

**Comment by Lekensteyn** [comment]
Yeah, the error parsing can certainly get better. For now just logout and login again.
